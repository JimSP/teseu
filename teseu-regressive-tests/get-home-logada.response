path:get-home-logada.request
error:Minotaur(super=br.com.cafebinario.teseu.model.Minotaur: get-home-logada.request.request contains error in expression N=$get-home-logada.responseBody.flIago, actual=S, expected=N)
