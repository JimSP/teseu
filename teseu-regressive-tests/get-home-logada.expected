200=#{context.get("get-home-logada.httpStatus")}